docs/resources/azure_express_route_circuits.md

```diff
@@ -0,0 +1,125 @@
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+
+Also, refer to [Azure documentation](https://docs.microsoft.com/en-us/rest/api/expressroute/express-route-circuits/list) for  properties available.
+Any attribute in the response may be accessed with the key names separated by dots (`.`).
+## Syntax
+
+An `azure_express_route_circuits` resource block returns all Azure express route circuits, either within a Resource Group (if provided)
+```ruby
+describe azure_express_route_circuits(resource_group: 'my-rg') do
+  
+end
+```
+
+## Properties
+
+|Property       | Description                                                                          | Filter Criteria<superscript>*</superscript> |
+|---------------|--------------------------------------------------------------------------------------|-----------------|
+| names          | A list of name  the resource group in which to create the ExpressRoute circuit. Changing this forces a new resource to be created. `MyResourceGroup`                                                 | `name`            |
+| ids            | A list of id the ExpressRoute circuit. Changing this forces a new resource to be created. `Myexpress circuitHostName                                                       | `id`              |
+| tags           | A list of `tag:value` pairs defined on the resources.                               | `tag`             |
+| provisioning_states             | State of express_route_circuits creation                                      | `provisioning_state`         |
+| types             |   Types of all the express_route_circuits | `type` |
+| locations           | Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.                                                 | `location`            |
+| service_provider_properties_bandwidth_in_mbps            | A list of The bandwidth in Mbps of the circuit being created on the Service Provider.                                                        | `id`              |
+| service_provider_properties_peering_locations           | A list of The name of the peering location and not the Azure resource location. Changing this forces a new resource to be created.                               | `tag`             |
+| service_provider_properties_names             |   The name of the ExpressRoute Service Provider. Changing this forces a new resource to be created.   | `type` |
```

Comment:
wrong fields actually

Reply:
Fields are correct but naming has extra word properties I kept .That was creating confusion.I have changed.

Reply:
Let me know If you are talking about any other fields.